.goreleaser-template.yaml

```diff
@@ -788,6 +788,10 @@ archives:
     format_overrides:
       - goos: windows
         format: zip
+    files:
```

Comment:
https://goreleaser.com/customization/archive/ defaults to including both `CHANGELOG*` and `changelog*`, the latter is a directory in our case.